salt/modules/disk.py

```diff
@@ -191,6 +202,8 @@ def percent(args=None):
         cmd = 'df -P'
     elif __grains__['kernel'] == 'OpenBSD':
         cmd = 'df -kP'
+    elif __grains__['kernel'] == 'AIX':
+        cmd = 'df -kP'
```

Comment:
yes, toyed with whether should or not, you'll notice one of the other changes has a combination. Will change it